src/Jackett.Common/Definitions/animeworld.yml

```diff
@@ -52,83 +35,155 @@ settings:
+        - name: re_replace
+          args: ["(?i)(minutit|minutter|minuti|minuty|minutos|минуты|минут|Minuten|минути|minuten|minuter)", "minutes"]
+        - name: re_replace
+          args: ["(?i)(dakika|minut|minuto|minuta|minutt|минута|Minute|minuut|分钟|分)", " minute"]
+        - name: re_replace
+          args: ["(?i)(tundi|timer|ore|godziny|horas|hodiny|hoden|часа|часов|ore|heures|Stunden|timmar)", "hours"]
+        - name: re_replace
+          args: ["(?i)(saat|tund|time|ora|godzina|hora|hodina|час|oră|heure|Stunde|uur|小时|時間|timme)", " hour"]
+        - name: re_replace
+          args: ["(?i)(päeva|dage|giorni|dni|dias|dny|дня|дней|zile|días|jours|Tagen|дни|dagen|dagar)", "days"]
+        - name: re_replace
+          args: ["(?i)(gün|päev|dag|giorno|dzień|dia|den|день|zi|día|jour|Tag|ден|天|日)", " day"]
+        - name: re_replace
+          args: ["(?i)(nädalat|uger|settimane|tygodnie|uker|semanas|týdny|недели|недель|săptămâni|semaines|Wochen|седмици|weken|veckor)", "weeks"]
+        - name: re_replace
+          args: ["(?i)(hafta|nädal|uge|settimana|tydzień|uke|semana|týden|неделю|săptămână|semaine|Woche|седмица|周|週間|vecka)", " week"]
+        - name: re_replace
+          args: ["(?i) (ay)", "month"]
+        - name: re_replace
+          args: ["(?i)(kuud|måneder|mesi|miesiące|meses|měsíce|месяца|месяцев|luni|meses|mois|Monaten|месеца|maanden|månader)", "months"]
+        - name: re_replace
+          args: ["(?i)(kuu|måned|mese|miesiąc|mês|měsíc|месяц|lună|mes|Monat|месец|maand|个月|ヶ月|månad)", " month"]
+        - name: re_replace
+          args: ["(?i)(aastat|anni|lata|anos|roky|года|ani|años|ans|Jahren|години)", " years"]
+        - name: re_replace
+          args: ["(?i)(yil|aasta|år|anno|rok|ano|год|año|Jahr|година|jaar|年)", " year"]
+        - name: re_replace
+          args: ["(?i) (an)", "year"]
+        - name: re_replace
+          args: ["(?i)(För |und)", ""] # Misc removals
+        - name: timeago
     downloadvolumefactor:
       case:
-        img[src="/pic/free.gif"]: 0
+        i[class*="fa-id-badge text-orange"]: 0 # 24 Hour FreeLeech From BON Store
+        i[class*="fa-trophy text-purple"]: 0 # Special FreeLeech For Certain User Groups
+        i[class*="fa-star text-bold"]: 0 # Freeleech From Token
+        i[class*="fa-coins text-bold"]: 0 # Freeleech From Token
+        i[class*="fa-globe text-blue"]: 0 # Global Freeleech
+        i[class*="fa-star text-gold"]: 0 # Freeleech
+        i[class*="fa-certificate text-pink"]: 0 # Featured Torrent
         "*": 1
     uploadvolumefactor:
-      text: 1
-# engine tbd
+      case:
+        i[class*="fa-gem text-green"]: 2 # Single Torrent Double Upload
+        i[class*="fa-globe text-green"]: 2 # Global Double Upload
+        i[class*="fa-certificate text-pink"]: 2 # Featured Torrent
+        "*": 1
+    minimumseedtime:
+      # 7 day (as seconds = 7 x 24 x 60 x 60)
+      text: 604800
+# UNIT3D 5.3.0
```

Comment:
Unit3D 5.3.0 was released only 3 days ago. 
Is the site's torrents page using 
`<script src="/js/unit3d.js?id=66d85a63faa080f39244" crossorigin="anonymous"></script>`
near the bottom of their torrents HTML or is the **id** different?

Reply:
Yes that's the correct ID. I also asked a admin, which stated that they pulled the newest version 2 days ago, which is 5.3.0 from my research